remote-trait-object/src/service.rs

```diff
@@ -32,13 +32,6 @@ pub type MethodId = u32;
 #[derive(PartialEq, Serialize, Deserialize, Debug, Clone, Copy)]
 pub struct HandleToExchange(pub(crate) ServiceObjectId);
 
```

Comment:
How about changing the commit title from "Remove new_singleton()" to "Remove unused new_singleton()"?

Reply:
done